patterns/ecs-v1/redis

```diff
@@ -1,3 +1,3 @@
 REDISTIMESTAMP %{MONTHDAY} %{MONTH} %{TIME}
-REDISLOG \[%{POSINT:pid}\] %{REDISTIMESTAMP:timestamp} \* 
-REDISMONLOG %{NUMBER:timestamp} \[%{INT:database} %{IP:client}:%{NUMBER:port}\] "%{WORD:command}"\s?%{GREEDYDATA:params}
+REDISLOG \[%{POSINT:[process][pid]}\] %{REDISTIMESTAMP:timestamp} \*
+REDISMONLOG %{NUMBER:timestamp} \[%{INT:[redis][database][id]} %{IP:[client][ip]}:%{NUMBER:[client][port]}\] "%{WORD:[redis][command][name]}"\s?%{GREEDYDATA:[redis][command][args]}
```

Comment:
TODO `client.port` type should be changed to `POSINT` as ECS assumes `type: long` for the field